frontend/src/views/ensemble/SingleNote.tsx

```diff
@@ -1,15 +1,24 @@
-import { NoteType } from "common/dist";
+import { Instrument, NoteType } from "common/dist";
 import { FC, useEffect, useState } from "react";
 import { Box } from "@mui/material";
 import { useRecoilValue } from "recoil";
 import { beatNumberAtom } from "../../recoil/beat";
-import { Synth } from "tone";
+import * as Tone from "tone";
+import { currentInstrumentAtom } from "../../recoil/instrument";
+import {
+  altoSaxSampler,
+  bassSampler,
+  fluteSampler,
+  guitarSampler,
+  marimbaSampler,
+  tubaSampler,
+} from "./Samplers";
 
 // Colors for notes
 const colorMapping = {
   [NoteType.REST]: "lightgray",
   [NoteType.ATTACK]: "gray",
-  [NoteType.SUSTAIN]: "darkgray", // ???? why is this lighter than gray
+  [NoteType.SUSTAIN]: "darkgray", // This is lighter than gray for some reason
```

Comment:
At this point, we might as well just record hex values for clarity. 

"#777" "#999" "#bbb"

Reply:
Will probably leave this here for now until I implement a palette picker for the user.